docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md

```diff
@@ -125,6 +125,30 @@ You will also need a table where the conversion events are stored. If you use th
 To fully finish the config you could overwrite the `snowplow__conversion_clause` variable in your project.yml, in case you would want to filter on specific types of conversions. You should also set the `snowplow__conversion_stitching` variable to true, if you have stitched_user_id in your conversions table.
 :::
 
+In case you store your conversion events elsewhere in the data warehouse (e.g. because those are transactional data that you decide you do not want to process through Snowplow), you can mutate that source table into the [format of the conversions table](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/conversions/#2-enabling_conversions_in_the_conversions_module) generated by the unified package, store it as a sql view, and refer to that in the variable `snowplow__conversion_source` (in the format of `schema.view_name`). 
```

Comment:
Needs to be an index.md link please

Reply:
can't get numbered ones to actually take you to that section but a quick scroll and they are there anyway